### ISSUE TITLE
training, the gradient disappears

### ISSUE THREAD

**Issue by mrswang1** [mit-han-lab/efficientvit#5]
When I finetune the model for my own dataset, the model has not yet started training, and the output of stage 3 and stage 4 of the model is NaN.

How can I solve this problem?

**Comment by arnvsnigi** [comment]
@mrswang1 Do you have the training code for efficientvit that may be you have written.I am a UG student and I wanted to study it.If possible please provide me with it.Your help would really help me a lot for my research.

**Comment by han-cai** [comment]
Hi mrswang1,

The training guide for ImageNet classification is available at https://github.com/mit-han-lab/efficientvit/blob/master/TRAINING.md.

Does the issue still exist? If so, can you give me an example for debugging?

Thanks,
Han
